src/openzaak/components/catalogi/tests/admin/test_import_export_catalogus.py

```diff
@@ -350,3 +350,42 @@ def test_import_catalogus_already_exists(self):
             response.text,
         )
         self.assertEqual(Catalogus.objects.count(), 1)
+
+    @override_settings(ALLOWED_HOSTS=["openzaak"])
+    def test_import_export_catalogus_different_hostname(self):
+        self.app.extra_environ["HTTP_HOST"] = "openzaak"
+
+        catalogus = CatalogusFactory.create(
+            rsin="000000000",
+            domein="TEST",
+            contactpersoon_beheer_naam="bla",
+            contactpersoon_beheer_telefoonnummer="0612345678",
+            contactpersoon_beheer_emailadres="test@test.nl",
+        )
+
+        url = reverse("admin:catalogi_catalogus_change", args=(catalogus.pk,))
+
+        response = self.app.get(url)
```

Comment:
I wouldn't do this as a `webtest`, but rather just use `call_command(...)` instead.